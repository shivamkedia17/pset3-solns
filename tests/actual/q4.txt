Initial State: 
Filter: 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0

Inserting: 1234
Filter: 0, 0, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 1, 0, 0, 1, 0, 1, 0, 0
Inserting: 63
Filter: 1, 1, 0, 0, 0, 0, 1, 1, 0, 0, 0, 0, 0, 1, 0, 1, 0, 0, 1, 0, 1, 0, 0
Inserting: 59
Filter: 1, 1, 1, 0, 0, 1, 1, 1, 0, 0, 0, 1, 0, 1, 0, 1, 0, 1, 1, 0, 1, 0, 0
Inserting: 62
Filter: 1, 1, 1, 1, 0, 1, 1, 1, 0, 0, 0, 1, 0, 1, 0, 1, 0, 1, 1, 0, 1, 0, 0

63 is PROBABLY in the filter.
59 is PROBABLY in the filter.
16 is NOT in the filter.
68 is NOT in the filter.

Inserting: 234
Filter: 1, 1, 1, 1, 0, 1, 1, 1, 0, 1, 0, 1, 0, 1, 0, 1, 0, 1, 1, 0, 1, 0, 1

98 is NOT in the filter.
62 is PROBABLY in the filter.
1234 is PROBABLY in the filter.
Final State: 
Filter: 1, 1, 1, 1, 0, 1, 1, 1, 0, 1, 0, 1, 0, 1, 0, 1, 0, 1, 1, 0, 1, 0, 1

