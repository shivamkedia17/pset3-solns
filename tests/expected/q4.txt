Initial State: 
Filter: 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0

Inserting: 1234
Filter: 1, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0
Inserting: 78
Filter: 1, 0, 0, 0, 1, 1, 1, 0, 0, 0, 0, 0, 0, 0, 0, 1, 0, 0, 0, 1, 0, 0, 0
Inserting: 26
Filter: 1, 0, 1, 0, 1, 1, 1, 0, 0, 0, 0, 0, 0, 0, 1, 1, 0, 0, 0, 1, 1, 0, 0
Inserting: 59
Filter: 1, 0, 1, 0, 1, 1, 1, 0, 0, 0, 0, 0, 1, 0, 1, 1, 0, 0, 0, 1, 1, 0, 0

78 is PROBABLY in the filter.
26 is PROBABLY in the filter.
42 is NOT in the filter.
20 is NOT in the filter.

Inserting: 234
Filter: 1, 0, 1, 1, 1, 1, 1, 0, 0, 0, 0, 0, 1, 0, 1, 1, 1, 0, 0, 1, 1, 0, 0

98 is NOT in the filter.
59 is PROBABLY in the filter.
1234 is PROBABLY in the filter.
Final State: 
Filter: 1, 0, 1, 1, 1, 1, 1, 0, 0, 0, 0, 0, 1, 0, 1, 1, 1, 0, 0, 1, 1, 0, 0

